config/smartyadmin.config.inc.php

```diff
@@ -75,6 +75,30 @@ function smartyTranslate($params, $smarty)
         $sprintf['legacy'] = $htmlEntities ? 'htmlspecialchars': 'addslashes';
     }
 
+    if ($isInPDF) {
+        return Translate::smartyPostProcessTranslation(
+            Translate::getPdfTranslation(
+                $params['s'],
+                $sprintf
+            ),
+            $params
+        );
+    }
+
+    // If the template is part of a module
+    if ($isInModule) {
+        return Translate::smartyPostProcessTranslation(
```

Comment:
This method has been removed IIRC, use `postProcessTranslation`

Reply:
@FabienPapet it's not part of this PR. It's due to the force push of @matthieu-rolland. I'll cherry pick and it would be solved